docs/writing-policies/spec/host-capabilities/03-container-registry.md

```diff
@@ -79,3 +79,113 @@ the payload would be the following ones:
 * Input payload: `"busybox:latest"`
 * Output payload: `{ "digest": "sha256:69e70a79f2d41ab5d637de98c1e0b055206ba40a8145e7bddb55ccc04e13cf8f"}`
 
+## OCI Image index
+
+This function fetches OCI images index manifest. This information can
+be used to identify specific images manifests, for one or more platforms.
```

Comment:
```suggestion
This function fetches OCI image index manifests. This information can
be used to identify specific image manifests, for one or more platforms.
```

Just a suggestion, I hope I'm not changing the meaning, but I think it reads a little better.